sdv/sampling/hierarchical_sampler.py

```diff
@@ -68,7 +67,7 @@ def _sample_rows(self, synthesizer, num_rows=None):
                 Sampled rows, shape (, num_rows)
         """
         num_rows = num_rows or synthesizer._num_rows
-        return synthesizer._sample_batch(int(num_rows), keep_extra_columns=True)
+        return synthesizer._sample_batch(num_rows, keep_extra_columns=True)
```

Comment:
As the docstring says (and I double checked), it's always int here. We either do this or change the docstring.